src/components/common/header/PageHeader.tsx

```diff
@@ -68,35 +74,47 @@ function PageHeader({
             console.error('Error in fetching server info')
         }
     }
+
+    useEffect(() => {
+        const expDate = localStorage.getItem('clickedOkay')
+        setExpiryDate(+expDate)
```

Comment:
No need for const. Use directly.
Also, check for undefined values as it'll set NaN.